front/app/modules/commercial/project_folders/citizen/containers/ProjectFolderShowPage/ProjectFolderDescription.tsx

```diff
@@ -217,6 +214,11 @@ const ProjectFolderDescription = memo<Props & InjectedIntlProps>(
                 </CollapseButtonWrapper>
               )}
           </Description>
+          {!isNilOrError(projectFolderFiles) &&
```

Comment:
These files were hidden, but not within QuillEditedContent, so the fix in this branch did not work. 

This change makes them visible at all times, which I think is acceptable. Basically, it will look like this now. A user can `show more` to view the hidden text, but the files will always be visable and tabbable. 


![Screen Shot 2022-04-12 at 09 15 49](https://user-images.githubusercontent.com/3614128/162912817-f6f00d1b-702d-454a-b569-b4c07c332a52.png)